notebooks/datasets/nldas_time_series.ipynb

```diff
@@ -0,0 +1,919 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # Create a function that would search for a data collection in the US VEDA STAC API
</pre><p>Why does this say US VEDA STAC API. What is the US here. I don't see any geographical information. This should likely be a chapter right?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NASA-IMPACT/veda-docs/pull/154/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>